Makefile

```diff
@@ -443,9 +443,12 @@ release: require-version require-s3cmd require-gh-token require-wget require-rub
 	git add lantern-$$VERSION-$$TAG_COMMIT* && \
 	(git commit -am "Latest binaries for Lantern $$VERSION ($$TAG_COMMIT)." && git push origin master) || true
 
-update-icons:
+update-resources:
```

Comment:
Renaming `update-icons` into `update-resources` to make it more generic and to be able to use it for building the status_pages directory.